USAGE.md

```diff
@@ -10,36 +10,34 @@ directory. The current directory should look like this:
 └── svc.proto
 ```
 
-Then, you'd run truss on your service definition file, like this: `truss svc.proto`.
-After running truss on your service, two new directories should be created in
-your current directory. One is called `third_party`, and the other is called
-`NAME-service`, where NAME is the name of the package defined in your
-definition file. Here's what that structure would look like:
+Then, you'd run truss on your service definition file, like this: `truss
+svc.proto`.  After running truss on your service, `NAME-service` should be
+created in your current directory. `NAME-service`, where NAME is the name of
+the package defined in your definition file. Here's what that structure would
+look like:
 
 ```
 .
 ├── NAME-service
-│   ├── docs
-│   │   └── docs.md
-│   ├── generated
-│   │   └── ...
-│   ├── handlers
-│   │   ├── client
-│   │   │   └── client_handler.go
-│   │   └── server
-│   │       └── server_handler.go
-│   ├── NAME-client
-│   │   └── client_main.go
-│   ├── NAME-server
-│   │   └── server_main.go
-│   └── svc.pb.go
+│   ├── docs
+│   │   └── docs.md
+│   ├── generated
+│   │   └── ...
+│   ├── handlers
+│   │   └── server
+│   │       └── server_handler.go
+│   ├── NAME-cli-client
+│   │   └── client_main.go
+│   ├── NAME-server
+│   │   └── server_main.go
```

Comment:
I see no reason why not. Do you see any reason why it is this way in the first place?